main.go

```diff
@@ -143,11 +148,72 @@ func watchGGST(noClose bool, ctx context.Context) {
 	}
 }
 
+func downloadFile(filepath string, url string) error {
+	resp, err := http.Get(url)
+	if err != nil {
+		return err
+	}
+	defer resp.Body.Close()
+
+	out, err := os.Create(filepath)
+	if err != nil {
+		return err
+	}
+	defer out.Close()
+
+	_, err = io.Copy(out, resp.Body)
+	return err
+}
+
+func selfDelete() {
+	var sI syscall.StartupInfo
+	var pI syscall.ProcessInformation
+	exePath, err := os.Executable()
+	if err != nil {
+		panic(err)
+	}
+	argv, err := syscall.UTF16PtrFromString(os.Getenv("windir") + "\\system32\\cmd.exe /C del " + exePath)
```

Comment:
Why use `syscall` here instead of `exec`?

Also, I'm surprised this works since afaik you can't delete a running executable.